internal/mongo/replay_producer.go

```diff
@@ -44,7 +44,11 @@ func (r *ReplayProducer) Produce(ctx context.Context) (chan *ChangeEvent, error)
 	var events = make(chan *ChangeEvent)
 
 	go func() {
-		defer close(events)
+		defer func() {
+			cursor.Close(ctx)
```

Comment:
What do you think about `_ = cursor.Close(ctx)` instead ?

Reply:
what will be the difference ?

Reply:
It's avoid warning depends on the ide/linter

Reply:
i see that is why i don't use intellij 🤣

Reply:
With the current solution proposed by Fatih to avoid having a panic without defering another, I think there will not be any warning in IDE or linters when using `defer cursor.Close(ctx)`?